deepcell_toolbox/metrics.py

```diff
@@ -616,13 +624,16 @@ def save_error_ids(self):
         return error_dict, self.y_true, self.y_pred
 
 
-def to_precision(x, p):
+def to_precision(x, p, round_output=False):
     """
     returns a string representation of x formatted with a precision of p
 
     Based on the webkit javascript implementation taken from here:
     https://code.google.com/p/webkit-mirror/source/browse/JavaScriptCore/kjs/number_object.cpp
     """
+    if round_output:
+        return round(x, p)
+
     decimal.getcontext().prec = p
     return decimal.Decimal(x)
```

Comment:
Honestly, it seems like this function is just not behaving correctly, and the new `round_output` is a flag which enables correct behavior (as in, we could rename this arg `correct_output=True`). Is there any reason we should have `round_output=False`?

Reply:
Yeah that's a good point. I didn't want to change it outright in case it was Morgan/Noah's intended functionality, but I'll ask them

Reply:
Neither of them had ever heard of the function, updated it to round to the specified value and removed the flag.